tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_interval.py

```diff
@@ -0,0 +1,70 @@
+import os
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.vulnerability_detector import callback_detect_vulnerability_scan_sleeping
+from wazuh_testing.tools.time import time_to_human_readable, unit_to_seconds
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# variables
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_interval.yaml')
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+interval_values = [1,2,5]
+interval_units  = ['s','m','h','d']
+
+parameters = []
+metadata = []
+ids = []
+
+for value in interval_values:
+  for unit in interval_units:
+    parameters.append({'INTERVAL': f'{value}{unit}'})
+    metadata.append({'interval': f'{value}{unit}'})
+    ids.append(f'{value}{unit}')
+
+
+# Configuration data
+configurations = load_wazuh_configurations(
+    configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+# fixtures
+@pytest.fixture(scope='module', params=configurations, ids=ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+# Tests
+
+
+def test_interval(get_configuration, configure_environment, restart_modulesd):
+
+    """
+    Check if modulesd waits `interval` between one vulnerability detector scan and another.
+
+    Parameters
+    ----------
+    custom_callback : function
+        Callback that will be applied to the value of tags to apply.
+    custom_error_message : str
+        Custom error message that will be shown if the test fails.
```

Comment:
This should be removed from the docstring

Reply:
https://github.com/wazuh/wazuh-qa/pull/686/commits/74e2d13602a93d933124745c4c2c187959b5e8be